view/script/trial.js

```diff
@@ -31,15 +31,16 @@
 
         differenceValue = moment(oldtoDateValue).diff(moment(oldfromDateValue), 'days');
     });
-    /**
-    * In above code, old toDate and fromDate values are first stored on focussing into toDate and difference calculated.
-    * Later the  difference is added on change of fromDate to toDate value in order to get new toDate.
-    */
 
-    $('#toDate').on('focusout', function submitAction () {
-        document
-        .getElementById('check-compliance')
-        .submit();
+    $('#check-compliance').submit(function submitAction (event) {
+        event.preventDefault();
+        $.ajax({
+            type: 'GET',
+            data: $('#check-compliance').serialize()
+        })
+        .done(function successSubmit () {
+            alert("Updating Chart");
```

Comment:
This is the correct use of `jQuery` AJAX syntax.
But this is not how AJAX works.
When you do an AJAX to get a response, the response needs to be captured and used.
Here the response is being thrown away.